coredocument/coredocument.proto

```diff
@@ -111,13 +136,27 @@ message SignatureData {
 // Signature contains the entity ID, public key used and signature)
 message Signature {
   // `signature_id` is a composed key signer_id+public_key used
-  bytes signature_id = 1;
+  bytes signature_id = 1 [
+    (proofs.exclude_from_tree) = true,
+    (proofs.field_length) = 52
```

Comment:
if excluded from the tree, do we need to set the field length for this?

Reply:
I do not think we need it since it will be skipped, I can remove the constraint.